docs/generate.php

```diff
@@ -0,0 +1,131 @@
+<?php
```

Comment:
We already have a script to generate docs in `docs/tool` I think we should consolidate. Having two tools to generate the same documentation is weird.

Also about the new `toc.json` file. I'm fine with the new file but this makes the `root-manifest.json` file useless as it's generated and then used to generate `manifest.json`. We should just generate `manifest.json` directly.

Reply:
I agree - I plan to remove my PHP script before this is merged. I am time constrained right now so I decided to write it in a language that I was a little more familiar to me so I could get it done faster :)